.travis.yml

```diff
@@ -4,7 +4,13 @@ services:
   - postgresql
 
 before_script:
-  - psql -U postgres -c 'CREATE DATABASE chitter_test;'
+  - psql -c 'CREATE DATABASE chitter_test;' -U postgres
+  - psql -U postgres -d chitter_test -f db/migrations/02_setup_user_table.sql
+  - psql -U postgres -d chitter_test -f db/migrations/03_setup_peep_table.sql
+  - psql -c 'CREATE DATABASE chitter;' -U postgres
```

Comment:
dont need this for travis as he's only running the tests

Reply:
I think mine is set up wrong in that case, as I got an error when I didnt include the chitter database.